### DISCUSSION TITLE
SU8 :-(

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6830] [General]
Hi developer and simmer,

would be nice to write down the experinces with the new SU8 update. For my system and the FBW bus the SU7 was perfect! Smooth, stutterfree and no problems when flying longer legs ...

with the newest update, everything went worse again ... buhhuuuuuuuu...

can anyone point out what the problem might be?? my frames dropping down to 10 and everything is going crazy .... not flyable at this point ..

![grafik](https://user-images.githubusercontent.com/76531893/156358691-169f8302-3b55-48bb-baa7-c502e96c648a.png)

EDIT: it becomes more and more crazy and I had a CTD ...

![grafik](https://user-images.githubusercontent.com/76531893/156359515-02127539-213d-404e-92c1-c025a08fb218.png)

greets

**Comment by tracernz** [comment]
Give the experimental version a try and see if that improves things. It has a different PFD implementation that's much nicer on frames.

**Reply by CptBayer** [reply]
thanx for the hint .... I´ll give it a try ...

greets to NZ

**Reply by CptBayer** [reply]
Hi,

did a second flight after posting my pics above. Was much better and I hadn´t these crazy frame drops again. Don´t ask me what the reasons are... Might be the small update from Asobo/Microsoft after the SU8?!

did also a flight with the experimental, round about 3 hours leg. My first feelings are, that it was much smoother than the dev version! 

I guess Asobo/Microsoft did a change in the balancing between CPU and GPU. I am using also during my flights an undocked window for my hardware MCDU. With SU7 I had no red stripes at all, with or without undocked window. 

With SU8 I had these red stripes massiv with undocked MCDU window. In spite of the red stripes, the stuttering is not very massiv and it is flyable. Without undocked it is nearly without red stripes.

Pic with external MCDU window:

![grafik](https://user-images.githubusercontent.com/76531893/156755174-e7ca2143-bbd6-4a40-a4f8-09ff49ee4458.png)

Pic without external MCDU window:

![grafik](https://user-images.githubusercontent.com/76531893/156755253-43ec1f14-ca0d-4b45-ad6f-43516b50685f.png)

Another change I saw is, that the GPU usage is yellow with the undocked window and the value of ms is high >25. With SU7 it was green and a value of not more than 7 ms ... hmmmmm .... 

my system: AMD Ryzen 3800xt. RTX 3090. Asus Rog Strix 570-e. 32 GB RAM with 3200 MHz and M2 drives. Not so bad I think....

greets
